Library/Homebrew/utils/sorbet/stubs.rb

```diff
@@ -0,0 +1,85 @@
+# typed: false
+# frozen_string_literal: true
+
+# Stubs for `sorbet-runtime`, all taken from `sorbet/t` except for `T::Sig.sig`.
+#
+# @private
+module T
```

Comment:
If you don't want to reinvent the wheel, you can consider using Shopify's [`sorbet-runtime-stub`](https://github.com/Shopify/sorbet-runtime-stub) gem.

There's definitely pros and cons to using other people's code vs implementing something yourself, so I see both sides.

Reply:
Thanks, that's definitely preferable to maintaining this ourselves.